chromium_src/BUILD.gn

```diff
@@ -37,6 +38,13 @@ static_library("chrome") {
     "//chrome/browser/net/proxy_config_monitor.h",
     "//chrome/browser/net/proxy_service_factory.cc",
     "//chrome/browser/net/proxy_service_factory.h",
+    "//chrome/browser/predictors/preconnect_manager.cc",
+    "//chrome/browser/predictors/preconnect_manager.h",
+    "//chrome/browser/predictors/proxy_lookup_client_impl.cc",
+    "//chrome/browser/predictors/proxy_lookup_client_impl.h",
+    "//chrome/browser/predictors/resolve_host_client_impl.cc",
+    "//chrome/browser/predictors/resolve_host_client_impl.h",
+    "//chrome/browser/profiles/profile.h",
```

Comment:
I think we don't depend on all of these any more? profile.h definitely shouldn't be included, and I think we can also drop proxy_lookup_client and resolve_host_client.

Reply:
I removed profile.h.
preconnect manager needs proxy_lookup_client and resolve_host_client as it resolves host there
before making network requests,